packages/mobile/src/sentry/saga.ts

```diff
@@ -0,0 +1,7 @@
+import { takeEvery } from 'redux-saga/effects'
+import { Actions } from 'src/sentry/actions'
+import { initializeSentryUserContext } from 'src/sentry/Sentry'
+
+export function* sentrySaga() {
+  yield takeEvery(Actions.INITIALIZE_SENTRY_USER_CONTEXT, initializeSentryUserContext)
```

Comment:
Small nit: if we don't expect this to happen more than once, this could be just a `take`, meaning we won't fork off a saga 'thread' to watch for others